integration-fe/src/components/TableInstance/TableInstance.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component, PropTypes } from 'react';
+import DataRow from './DataRow';
+import '../../styles/main.less';
+import './styles/styles.less';
+
+class TableInstance extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      instances: [],
+    };
+    this.onConfigureNew = this.onConfigureNew.bind(this);
+  }
+
+  componentWillMount() {
```

Comment:
Is this really necessary? I understood that you created the instances in the state at Constructor, then you will update it at componentWillReceiveProps, am I right?